We should check /sbin and /usr/sbin earlier in the configure process.
If there is a binary named 'dump' that is not a filesystem dump command (which
is normally found in /usr/sbin) that occurs in the /usr/bin, then configure can
hang.

Gentoo-bug: 248838
Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>
diff -Nuar --exclude '*~' amanda-2.6.0p2.orig/config/amanda/progs.m4 amanda-2.6.0p2/config/amanda/progs.m4
--- amanda-2.6.0p2.orig/config/amanda/progs.m4	2008-02-15 13:35:57.000000000 -0800
+++ amanda-2.6.0p2/config/amanda/progs.m4	2008-11-25 17:46:56.879917150 -0800
@@ -24,7 +24,7 @@
 #
 AC_DEFUN([AMANDA_INIT_PROGS],
 [
-    SYSPATH="/bin:/usr/bin:/sbin:/usr/sbin:/opt/SUNWspro/bin:/usr/ucb:/usr/sfw/bin:/usr/bsd:/etc:/usr/etc"
+    SYSPATH="/sbin:/bin:/usr/sbin:/usr/bin:/opt/SUNWspro/bin:/usr/ucb:/usr/sfw/bin:/usr/bsd:/etc:/usr/etc"
     # expand prefix or exec_prefix in LOCPATH
     LOCPATH=`(
 	test "x$prefix" = xNONE && prefix=$ac_default_prefix
